swagger.yaml

```diff
@@ -10,33 +10,26 @@ basePath: "/v1"
 schemes:
   - http
 tags:
-  - name: "hello"
+  - name: "session"
 paths:
-  /hello:
-    get:
+  /session:
+    post:
       tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
+        - session
+      summary: Create a session endpoint
+      description: Creates a session and logs request back
       produces:
         - application/json
       responses:
-        200:
-          description: OK
+        201:
+          description: Created
           schema:
-            $ref: "#/definitions/HelloResponse"
-        500:
-          $ref: '#/responses/InternalError'
+            $ref: "#/definitions/SessionResponse"
+        400:
+          description: Bad Request
+          schema:
+            $ref: "#/responses/BadRequest"
 
 responses:
   InternalError:
-    description: "Failed to process the request due to an internal error"
-
-definitions:
-  HelloResponse:
-    type: object
-    properties:
-      message:
-        type: string
```

Comment:
Regarding the swagger spec - I would copy the raw file of swagger.yaml and paste it into https://editor.swagger.io/ (left hand side of web page) this will inform you of any errors in your swagger spec and also has a decent ui for reading the spec (right hand side of web browser)